jhub_apps/service/auth.py

```diff
@@ -9,7 +9,7 @@
 logger = structlog.get_logger(__name__)
 
 
-def create_access_token(data: dict, expires_delta: typing.Optional[timedelta] = None):
+def _create_access_token(data: dict, expires_delta: typing.Optional[timedelta] = None):
```

Comment:
I don't know if there is a BC policy in place, but this is BC breaking.

Reply:
Good point. Although we don't have a BC policy at the moment, but since this function is quite unlikely to be used by users and jhub-apps is in relatively early stages of development, I would rather skip the deprecation cycle for this one and add a task to define BC policy for future.